brake/backends/cachebe.py

```diff
@@ -51,12 +52,13 @@ def _keys(self, func_name, request, ip=True, field=None, period=None):
 
     def count(self, func_name, request, ip=True, field=None, period=60):
         """Increment counters for all relevant cache_keys given a request."""
-        counters = dict((key, 1) for key in self._keys(
+        counters = dict((key, (1, time.time() + period)) for key in self._keys(
             func_name, request, ip, field, period))
         counters.update(cache.get_many(counters.keys()))
-        for key in counters:
-            counters[key] += 1
-        cache.set_many(counters, timeout=period)
+        for key, value in counters.items():
+            count, expiration = value
+            count += 1
+            cache.set(key, (count, expiration), timeout=expiration - time.time())
```

Comment:
I can test it, but Travis isn't liking the tests on this branch, for some reason. It can't import unittest, even though there's nothing in this PR that would make that fail.